modules/rosa-adding-taints.adoc

```diff
@@ -9,112 +9,125 @@
-<2> Replace `<key>=<value>:<effect>,<key>=<value>:<effect>` with a key, value, and effect for each taint, for example `--taints=key1=value1:NoSchedule,key2=value2:NoExecute`. Available effects include `NoSchedule`, `PreferNoSchedule`, and `NoExecute`.This list overwrites any modifications made to node taints on an ongoing basis.
-+
-The following example adds taints to the `db-nodes-mp` machine pool:
-+
-[source,terminal]
-----
-$ rosa edit machinepool --cluster=mycluster --replicas 2 --taints=key1=value1:NoSchedule,key2=value2:NoExecute db-nodes-mp
-----
-+
-.Example output
-[source,terminal]
-----
-I: Updated machine pool 'db-nodes-mp' on cluster 'mycluster'
-----
-
-* To add or update taints for a machine pool that uses autoscaling, run the following command:
-+
-[source,terminal]
-----
-$ rosa edit machinepool --cluster=<cluster_name> \
-                        --min-replicas=<minimum_replica_count> \ <1>
-                        --max-replicas=<maximum_replica_count> \ <1>
-                        --taints=<key>=<value>:<effect>,<key>=<value>:<effect> \ <2>
-                        <machine_pool_id>
-----
-<1> For machine pools that use autoscaling, you must provide minimum and maximum compute node replica limits. If you do not specify the arguments, you are prompted for the values before the command completes. The cluster autoscaler does not reduce or increase the machine pool node count beyond the limits that you specify. If you deployed ROSA using a single availability zone, the `--min-replicas` and `--max-replicas` arguments define the autoscaling limits in the machine pool for the zone. If you deployed your cluster using multiple availability zones, the arguments define the autoscaling limits in total across all zones and the counts must be multiples of 3.
-<2> Replace `<key>=<value>:<effect>,<key>=<value>:<effect>` with a key, value, and effect for each taint, for example `--taints=key1=value1:NoSchedule,key2=value2:NoExecute`. Available effects include `NoSchedule`, `PreferNoSchedule`, and `NoExecute`.This list overwrites any modifications made to node taints on an ongoing basis.
-+
-The following example adds taints to the `db-nodes-mp` machine pool:
-+
-[source,terminal]
-----
-$ rosa edit machinepool --cluster=mycluster --min-replicas=2 --max-replicas=3 --taints=key1=value1:NoSchedule,key2=value2:NoExecute db-nodes-mp
-----
-+
-.Example output
-[source,terminal]
-----
-I: Updated machine pool 'db-nodes-mp' on cluster 'mycluster'
-----
+// ifdef::openshift-rosa[]
```

Comment:
There is quite a bit of new content in this module that you have commented out. If the intent is not to use it, I suggest removing it entirely instead of commenting it out.